app/feature/feature-help-center/src/main/kotlin/com/hedvig/android/feature/help/center/data/GetQuickLinksUseCase.kt

```diff
@@ -22,73 +20,86 @@ import octopus.AvailableSelfServiceOnContractsQuery
+            quickLinkDestination = QuickLinkDestination.OuterDestination.QuickLinkTravelCertificate,
             titleRes = R.string.HC_QUICK_ACTIONS_TRAVEL_CERTIFICATE,
             hintTextRes = R.string.HC_QUICK_ACTIONS_TRAVEL_CERTIFICATE_SUBTITLE,
           ),
         )
       }
-
-      if (featureManager.isFeatureEnabled(Feature.PAYMENT_SCREEN).first()) {
+      if (memberActionOptions.isCancelInsuranceEnabled) {
         add(
           QuickAction.StandaloneQuickLink(
-            quickLinkDestination = QuickLinkDestination.QuickLinkConnectPayment,
+            quickLinkDestination = QuickLinkDestination.OuterDestination.QuickLinkTermination,
+            titleRes = R.string.HC_QUICK_ACTIONS_CANCELLATION_TITLE,
+            hintTextRes = R.string.HC_QUICK_ACTIONS_CANCELLATION_SUBTITLE,
+          ),
+        )
+      }
+      if (memberActionOptions.isConnectPaymentEnabled) {
+        add(
+          QuickAction.StandaloneQuickLink(
+            quickLinkDestination = QuickLinkDestination.OuterDestination.QuickLinkConnectPayment,
             titleRes = R.string.HC_QUICK_ACTIONS_PAYMENTS_TITLE,
             hintTextRes = R.string.HC_QUICK_ACTIONS_PAYMENTS_SUBTITLE,
           ),
         )
       }
+      if (memberActionOptions.isMovingEnabled) {
+        add(
+          QuickAction.StandaloneQuickLink(
+            quickLinkDestination = QuickLinkDestination.OuterDestination.QuickLinkChangeAddress,
+            titleRes = R.string.HC_QUICK_ACTIONS_CHANGE_ADDRESS_TITLE,
+            hintTextRes = R.string.HC_QUICK_ACTIONS_CHANGE_ADDRESS_SUBTITLE,
+          ),
+        )
+      }
+      if (memberActionOptions.firstVetAction?.sections?.isNotEmpty() == true) {
+        add(
+          QuickAction.StandaloneQuickLink(
+            quickLinkDestination = QuickLinkDestination.InnerHelpCenterDestination.FirstVet(
+              sections = memberActionOptions.firstVetAction.sections,
+            ),
+            titleRes = R.string.HC_QUICK_ACTIONS_FIRSTVET_TITLE,
+            hintTextRes = R.string.HC_QUICK_ACTIONS_FIRSTVET_SUBTITLE,
+          ),
+        )
+      }
+      if (memberActionOptions.sickAbroadAction?.partners?.isNotEmpty() == true) {
+        add(
+          QuickAction.StandaloneQuickLink(
+            quickLinkDestination = QuickLinkDestination.InnerHelpCenterDestination.QuickLinkSickAbroad(
+              emergencyNumber = memberActionOptions.sickAbroadAction.partners[0].phoneNumber,
+            ),
+            titleRes = R.string.HC_QUICK_ACTIONS_SICK_ABROAD_TITLE,
+            hintTextRes = R.string.HC_QUICK_ACTIONS_SICK_ABROAD_SUBTITLE,
+          ),
+        )
+      }
+      if (memberActionOptions.isEditCoInsuredEnabled) {
+        val contracts = apolloClient.query(AvailableSelfServiceOnContractsQuery())
+          .safeExecute()
+          .toEither(::ErrorMessage)
+          .onLeft { logcat(LogPriority.ERROR) { it.message ?: "Could not fetch common claims" } }
```

Comment:
A little trick here which I've found useful
```suggestion
          .onLeft { logcat(LogPriority.ERROR) { "Could not fetch common claims ${it.message}" } }
```

This may look odd at first glance but the idea is that if the message is in fact not null, then on the logs in DDog we'll get the error message but we'll not be able to easily track it back to which log in the app it actually was. With the string always being there it's a quick "search all" and we're exactly at the right spot!